src/styles/blocks/watch.scss

```
@@ -0,0 +1,80 @@
+.watch {
+  position: relative;
+
+  display: flex;
+  justify-content: center;
+
+  @include size ($watch-size);
+
+  background-color: #87ceeb;
+  background-image: linear-gradient(#87ceeb, gold);
+  box-shadow: 20px 20px 20px rgba(0,0,0,0.25);
+  border-radius: 50%;
+  border: 3px solid black;
+
+  &__center {
+    position: relative;
+    align-self: center;
+    z-index: 100;
+
+    width: 15px;
+    height: 15px;
+
+    border-radius: 50%;
+    background-color: white;
+    border: 2px solid black;
+  }
+
+  &__hand {
+    position: absolute;
+    top: 75px;
```

your hour hand is not spinning, you need to add a property: animation
```suggestion
  &__hand {
  	animation: watch 3600s linear infinite;
    transform-origin: bottom;
    
    position: absolute;
    top: 75px;
```

---

was created, thanks.